lib/deltachat.ts

```diff
@@ -25,25 +26,28 @@ const DC_SHOW_EMAILS = [
   C.DC_SHOW_EMAILS_OFF,
 ]
 interface NativeContext {}
+interface NativeAccount {}
 
 /**
  * Wrapper around dcn_context_t*
  */
 export class DeltaChat extends EventEmitter {
   dcn_context: NativeContext
```

Comment:
this doesn't seem needed anymore

Reply:
It's needed for the context, but yes can get moved/deleted